### DISCUSSION TITLE
连接耗时太久

### DISCUSSION THREAD

**Discussion by 13600100046** [getlantern/lantern-forum-cn#667] [Q&A]
我不知道你们公司是怎么平衡一个用户30s内没有使用流量后，是否关闭该用户的vpn连接。每次超过30s没有使用流量，就会自动断网，什么时候能连接上则看你们的用户拥挤度，同时段用户少的情况下，30s内就能重连上，而这几天，是你们的用户量暴增了么，每次把我的网络断开，我重连都要超过1min，有时甚至达到5min，属实有点离谱了，体验感很差

**Comment by MaxMKMax** [comment]
@13600100046 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Comment by MaxMKMax** [comment]
@13600100046 您好. 请您提供我报告票号. 有了票号我们可以更容易找到问题然后解决. 谢谢.
